main.go

```diff
@@ -70,61 +70,76 @@ func main() {
 	}
 }
 
-func check(e error) {
-	if e != nil {
-		panic(e)
-	}
+func get_kd_files(index int) (string, string) {
+	var keyfile = db + "/" + strconv.Itoa(index) + ".key"
+	var datafile = db + "/" + strconv.Itoa(index) + ".data"
+	return keyfile, datafile
 }
 
 func backup_etcd(ep string, dir string) error {
 	ctx := context.Background()
 	cli, err := clientv3.New(clientv3.Config{
 		Endpoints: []string{ep},
 	})
-	check(err)
+	if err != nil {
+		return err
+	}
 	defer cli.Close()
+
 	resp, err := cli.Get(ctx, "", clientv3.WithPrefix(), clientv3.WithSort(clientv3.SortByKey, clientv3.SortDescend))
-	check(err)
+	if err != nil {
+		return err
+	}
 
 	err = os.Mkdir(db, 0700)
-	check(err)
+	if err != nil {
+		return err
+	}
 
 	for i, ev := range resp.Kvs {
 		logrus.Debugf("%d) %s\n", i, ev.Key)
-		var keyfile = db + "/" + strconv.Itoa(i) + ".key"
-		var datafile = db + "/" + strconv.Itoa(i) + ".data"
+		keyfile, datafile := get_kd_files(i)
```

Comment:
nit; `get_kd_files` is more Pythonic in style. In Go code camelCase is preferred. I'd go with the suggestion, but up to you to decide something better.

```suggestion
		keyFile, dataFile := getFileNamesForKey(i)
```